include/mbedtls/aes.h

```diff
@@ -87,6 +87,7 @@ typedef struct mbedtls_aes_context
 {
     int nr;                     /*!< The number of rounds. */
     uint32_t *rk;               /*!< AES round keys. */
+    uint32_t *frk;              /*!< Fake AES round keys. */
```

Comment:
fake key should be warped with MBEDTLS_AES_SCA_COUNTERMEASURES flag

Reply:
This flag is about side channel attacks, not about fault injection - should we provide a different flag?

Reply:
Ok. Let's commit it without flag. I will create "low priority" task to add FI flag to baremetal branch